__tests__/app.test.js

```diff
@@ -356,4 +358,32 @@ describe("ENDPOINT TESTING", () => {
         });
     });
   });
+
+  describe("/api/articles/:article_id/comments", () => {
+    test("POST: 201, Post a comment to specied article with, request being a username and a body, returning the posted comment", () => {
+      const newComment = {
+        username: "butter_bridge",
+        body: "Relatable content!",
+      };
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send(newComment)
+        .expect(201)
+        .then((response) => {
+          expect(response.body.comment).toBe("Relatable content!");
```

Comment:
The comment that your server responds with should be an object with a few properties on there rather than just the comment body itself - take a look back at the trello ticket for a reminder 👍